task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskFormDatePickers.vue

```diff
@@ -146,6 +149,14 @@ export default {
     };
   },
   computed: {
+    checkDate() {
+      const today = new Date();
+      const tomorrow = new Date();
+      const nextWeek = new Date();
+      tomorrow.setDate(tomorrow.getDate() + 1);
+      nextWeek.setDate(nextWeek.getDate() + 7);
+      return { today, tomorrow, nextWeek };
+    },  
```

Comment:
Instead of creating an object, I would prefer something like this : 
```suggestion
    checkDate(days) {
      if (!days) {
         days=0;
      }
      const date = new Date();
      date.setDate(date.getDate() + days);
      return date;
    },  
```
and you call with checkDate() for checking today, checkDate(1) for checking tomorrow, checkDate(7) for checking next week